research-software-directory/RSD-as-a-service
# Improve SonarCloud actions

# SonarCloud improvements

Changes proposed in this pull request:

* The single SonarCloud workflow is now splitted over four workflows that only run when relevant code has changed
* Upload code coverage reports to SonarCloud in the workflows

How to test :

* Inspect the new and changed workflow files
* Check that the actions ran correctly on this PR

Closes #965

PR Checklist:
